autoinstrumentation-dotnet/Dockerfile

```diff
@@ -13,8 +13,8 @@ LABEL name="Autoinstrumentation-dotnet" \
     vendor="Sumo Logic" \
     version="${UPSTREAM_VERSION}" \
     release="${RELEASE}" \
-    summary="UBI based OpenTelemetry .NET Automatic Instrumentation " \
-    description="Kubernetes Operator for OpenTelemetry Collector" \
+    summary="OpenTelemetry .NET Automatic Instrumentation" \
+    description="UBI based OpenTelemetry .NET Automatic Instrumentation" \
     maintainer="opensource-collection-team@sumologic.com"
```

Comment:
This is UBI-based and the other ones are not? I don't understand whether it's deliberate or a typo.

Reply:
I see now that the other ones are also UBI-based. Please stick to one naming convention.

Reply:
I don't think this information (`UBI-based`) is usable for end-users